tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import wknml
```

Comment:
I don't really understand the purpose of this file. What is the difference to `test_snapshot_readandwrite.py`? Due to the lodashes in the methods, it won't be picked up by pytest (that's why it has the `if __name__ ...` code?). However, it isn't executed on the CI?

Reply:
The methods in this file are meant to be executed when there were changes in the behaviour of the nml parsing, so you can generate new (pickled/unpickled) files to compare against.

Reply:
Ah ok, then I'd suggest to rename the file to `refresh_snapshots.py`.